scripts.py

```diff
@@ -220,6 +541,7 @@ def fix_identifier_with_equivalents(self, primary_identifier, equivalent_ids):
 
         equivalent_ids = primary_identifier.equivalent_identifier_ids(
             levels=6, threshold=0)
+        # TODO: equivalencies is not used?
```

Comment:
I think originally there was a nested loop, where instead of just looking at `primary_identifier` we looked at every identifier in `equivalent_ids`. You can take this out if you like.